vk_video_encoder/libs/VkVideoEncoder/VkEncoderConfig.cpp

```diff
@@ -524,6 +525,19 @@ int EncoderConfig::ParseArguments(int argc, char *argv[])
         return -1;
     }
 
+    frameCount = inputFileHandler.GetFrameCount(input.width, input.height, input.bpp, input.chromaSubsampling);
```

Comment:
What if the user wanted to specify a frame count smaller than the full video length?

Reply:
OK, thank you, Stepahan!